config.py

```diff
@@ -65,4 +65,17 @@ def configure(advanced):
                     private=True))
 
 
+conf.registerGroup(Fedora, 'karma')
+conf.registerGlobalValue(
+    Fedora.karma, 'db_path',
+    registry.String('/var/tmp/supybot-karma.db',
+                    """Path to a karma db on disk"""))
```

Comment:
supybot provides an interface to connect to the DB directly?
I'm not seeing any DB related code in this p-r.